### PR TITLE
clipboard 0.3.2 (new formula)

### PR DESCRIPTION
**Pull request by tranzystorekk**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/clipboard.rb
Comment (tranzystorek-io) [reply Formula/clipboard.rb:22]: I can run the project's shell script test suite, but it requires the binary to be in PATH, can that be done?

Comment (chenrui333) [reply Formula/clipboard.rb:22]: what binary?

Comment (tranzystorek-io) [reply Formula/clipboard.rb]: Added a small copy-paste test instead

**Comment by tranzystorek-io** [comment]
Hard to tell what's wrong with the linux test in CI, it passes locally for me.

**Comment by Slackadays** [comment]
That's definitely weird, maybe one of the maintainers can help?

**Comment by Slackadays** [comment]
@tranzystorek-io The latest Clipboard version might have fixed the issue here

File: Formula/clipboard.rb
Comment (SMillerDev) [inline Formula/clipboard.rb:27]: The prefix is set by the standard arguments
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
```

File: Formula/clipboard.rb
Comment (SMillerDev) [inline Formula/clipboard.rb:31]: Doesn't cmake install this?

File: Formula/clipboard.rb
Comment (tranzystorek-io) [reply Formula/clipboard.rb:27]: args set `CMAKE_INSTALL_PREFIX`, this package requires a custom INSTALL_PREFIX instead

Comment (SMillerDev) [reply Formula/clipboard.rb:27]: Could that be fixed upstream ?

File: Formula/clipboard.rb
Comment (tranzystorek-io) [reply Formula/clipboard.rb:31]: No, manpage needs to be installed manually

File: Formula/clipboard.rb
Comment (tranzystorek-io) [reply Formula/clipboard.rb:27]: A question to @Slackadays, but afaik it was explicitly switched in recent release.

File: Formula/clipboard.rb
Comment (SMillerDev) [reply Formula/clipboard.rb:31]: Could you make an issue for that upstream because that's a weird thing to leave out of your build system. Makes me think they just forgot about the manpages

Comment (tranzystorek-io) [reply Formula/clipboard.rb:31]: Was addressed in upstream recently, pending release

Comment (tranzystorek-io) [reply Formula/clipboard.rb]: Done

File: Formula/clipboard.rb
Comment (tranzystorek-io) [reply Formula/clipboard.rb]: Done

**Comment by Slackadays** [comment]
All checks have passed :)

**Review by SMillerDev — APPROVED** [review]

**Comment by tranzystorek-io** [comment]
@chenrui333 anything blocking this PR from getting merged?

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
> @chenrui333 anything blocking this PR from getting merged?
nope, sorry about the delay, merging now.

**Comment by chenrui333** [comment]
@tranzystorek-io, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**
